bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/formsportal/LinkImpl.java

```diff
@@ -56,6 +59,8 @@
     private static final String PN_PARAM_KEY = "key";
     private static final String PN_PARAM_VALUE = "value";
     public static final Logger logger = LoggerFactory.getLogger(LinkImpl.class);
+    private static final String QP_AF_DEFAULT_MODE_KEY = "wcmmode";
```

Comment:
You should make use of WCMMode class for these constants, since this is not owned by core components. For example, https://github.com/adobe/aem-core-cif-components/blob/b2cdd1470bea845031becaea6ef3c52a9d97f5d7/bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/servlets/SpecificPageFilterFactoryTest.java#L71

Reply:
okay. Thanks for the reference!